xapres_package/load.py

```diff
@@ -13,59 +13,63 @@
 import pandas as pd
 from tqdm import tqdm
 
-from utils import *
+from .utils import phase2range, sonify, dB, coherence, generate_range_diff
 
-sys.path.append(os.path.join(os.path.dirname(__file__), "lib"))
-
-def load_zarr(site = "A101", directory = "gs://ldeo-glaciology/apres/greenland/2022/single_zarrs_noencode/"):
+def load_zarr(site = "A101", 
+            directory = "gs://ldeo-glaciology/apres/greenland/2022/single_zarrs_noencode/"
+            ):
     """Load ApRES data stored in a zarr directory as an xarray and add functionality"""
     
     import numpy as np
     import xarray as xr
         
     ds = xr.open_dataset(directory + site,
-        engine = 'zarr', 
-        chunks = {}) 
-    
-    # add db function as new bound method of DataArrays
-    xr.DataArray.db = lambda self : 20*np.log10(np.abs(self))
+            engine = 'zarr', 
+            chunks = {}) 
     
+    # add the db function as new bound method of DataArrays
+    xr.DataArray.dB = dB
     
-    # add sonify function (only if soundfile and sounddevice are installed)
-    def sonify(self, play = True, save = False, wav_filename = "chirp"):
+    # add the sonify function as a bound method of DataArrays
+    xr.DataArray.sonify = sonify 
+     

```

Comment:
we could add something like
```
ds['chirp_time'] = ds.chirp_time.values.astype("float64")/1e9
```
here to deal with the results of a bug identified in #34